www/cryptoparty.html

```diff
@@ -55,6 +55,9 @@
 			a:hover {
 				color: #ffffff;
 			}
+			li {
```

Comment:
ersetze `li` durch .termine und füge in Zeile 77 in `ul` ein `class="termine"` ein